pkg/systemd/services.html

```diff
@@ -136,6 +138,134 @@
+      <div class="position-colon">:</div>
+      <input type="text" data-index="{{index}}" data-content="minutes" value="{{minutes}}" class="form-control">
+      <button class="btn btn-default dropdown-toggle fa fa-plus" data-content="add" data-index="{{index}}" type="button" ></button>
+      <button class="btn btn-default dropdown-toggle pficon-close" data-content="close" data-index="{{index}}" type="button" {{close}}></button>
+      <div data-content="hr-error" class="repeat-error" data-index="{{index}}"></div>
+      <div data-content="min-error" class="repeat-error" data-index="{{index}}"></div>
+    </div>
+  {{/repeat}}
+  </script>
+
+  <script id="repeat-weekly-tmpl" type="x-template/mustache">
+  {{#repeat}}
+    <div class="form-inline">
+      <div class="btn-group bootstrap-select form-control dropdown " data-index="{{index}}" data-content="week-days">
+        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="false">
+          <span class="pull-left" value="{{days_value}}" translatable="yes">{{days_text}}</span>
+          <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu">
+          <li value="1"><a>Monday</a></li>
+          <li value="2"><a>Tuesday</a></li>
+          <li value="3"><a>Wednesday</a></li>
+          <li value="4"><a>Thursday</a></li>
+          <li value="5"><a>Friday</a></li>
+          <li value="6"><a>Saturday</a></li>
+          <li value="7"><a>Sunday</a></li>
+        </ul>
+      </div>
+      <input type="text" value="{{hours}}" data-index="{{index}}" data-content="hours" class="form-control" >
+      <div class="position-colon">:</div>
+      <input type="text" data-index="{{index}}" data-content="minutes" value="{{minutes}}" class="form-control">
+      <button class="btn btn-default dropdown-toggle fa fa-plus" data-content="add" data-index="{{index}}" type="button" ></button>
+      <button class="btn btn-default dropdown-toggle pficon-close" data-content="close" data-index="{{index}}" type="button" {{close}}></button>
+      <div data-content="hr-error" class="repeat-error" data-index="{{index}}"></div>
+      <div data-content="min-error" class="repeat-error" data-index="{{index}}"></div>
+    </div>
+  {{/repeat}}
+  </script>
+
+  <script id="repeat-monthly-tmpl" type="x-template/mustache">
+  {{#repeat}}
+    <div class="form-inline">
+      <div class="btn-group bootstrap-select dropdown form-control" data-index="{{index}}" data-content="month-days">
+        <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="false">
+          <span class="pull-left" value="{{days_value}}" translatable="yes">{{days_text}}</span>
+          <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu vertical-scroll">
+          <li value="1"><a>1st</a></li>
```

Comment:
Good question. Yes, I think translatable=yes could work.
So how would it look in German? In Swedish it would be:
1:a, 2:a, 3:e, 4:e, 5:e etc. and maybe that's just asking for trouble.
1, 2, 3, 4, 5 etc. would work for me too.